proto/private/BUILD.release

```diff
@@ -54,7 +54,7 @@ proto_lang_toolchain(
 # Redirect everything else to the source
 redirect_targets = [
     "wrappers_proto",
-    "well_known_protos",
+    "well_known_types_proto",
```

Comment:
This seems to contain a two typos, it should be "well_known_type_protos".

To verify manually `blaze build //proto/private:all` should work.